.github/CODEOWNERS

```diff
@@ -0,0 +1,76 @@
+
+/.cargo                             @benesch
+/.config                            @benesch
+/.devcontainer                      @mjibson
+/.github                            @benesch
+/bin
+/ci                                 @benesch
+/doc/user                           @MaterializeInc/DevEx
+/doc/developer/reference/compute    @MaterializeInc/compute
+/doc/developer/reference/storage    @MaterializeInc/storage
+/ci                                 @MaterializeInc/qa
+/src/adapter                        @MaterializeInc/surfaces
+/src/alloc                          @benesch
+/src/audit-log                      @MaterializeInc/surfaces
+/src/avro                           @umanwizard
+/src/avro-derive                    @umanwizard
+/src/build-id                       @umanwizard
+/src/build-info                     @benesch
+/src/ccsr                           @benesch
+/src/cloud-resources                @MaterializeInc/cloud
+/src/clusterd                       @MaterializeInc/compute @MaterializeInc/storage
+/src/compute                        @MaterializeInc/compute
+/src/compute-client                 @MaterializeInc/compute
+/src/environmentd                   @MaterializeInc/surfaces
+/src/expr                           @MaterializeInc/compute
+/src/expr-test-util                 @MaterializeInc/compute
+/src/frontegg-auth                  @MaterializeInc/compute
+/src/http-util                      @MaterializeInc/surfaces
+/src/interchange                    @MaterializeInc/storage
+/src/kafka-util                     @MaterializeInc/storage
+/src/kinesis-util                   @MaterializeInc/storage
+/src/lowertest                      @MaterializeInc/compute
+/src/lowertest-derive               @MaterializeInc/compute
+/src/metabase                       @benesch
+/src/mz                             @MaterializeInc/surfaces
+/src/npm                            @benesch
+/src/orchestrator                   @benesch
+/src/orchestrator-kubernetes        @benesch
+/src/orchestrator-process           @benesch
+/src/orchestrator-tracing           @benesch
+/src/ore                            @benesch
+/src/persist                        @MaterializeInc/persist
+/src/persist-client                 @MaterializeInc/persist
+/src/persist-types                  @MaterializeInc/persist
+/src/pgcopy                         @MaterializeInc/surfaces
+/src/pgrepr                         @MaterializeInc/surfaces
+/src/pgtest                         @MaterializeInc/surfaces
+/src/pgwire                         @MaterializeInc/surfaces
+/src/pid-file                       @benesch
+/src/postgres-util                  @MaterializeInc/surfaces
+/src/prof                           @umanwizard
+/src/proto                          @aalexandrov
+/src/repr                           @MaterializeInc/storage @MaterializeInc/compute
```

Comment:
/src/repr/src/row to @MaterializeInc/persist please

Reply:
Done.